types/node/test/https.ts

```diff
@@ -0,0 +1,157 @@
+import * as http from 'node:http';
```

Comment:
Can you move all the https tests from `node-tests.ts` in here as well?

Reply:
[Done](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54886/commits/4e3e5525c9bd8698fb8b503265060b3c162e61bc) :white_check_mark: